docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -117,12 +114,11 @@ Office.context.ui.displayDialogAsync('https://myDomain/myDialog.html', {height:
 );
 ```
 
-> [!NOTE]
->
-> - Office passes an [AsyncResult](/javascript/api/office/office.asyncresult) object to the callback. It represents the result of the attempt to open the dialog box. It does not represent the outcome of any events in the dialog box. For more on this distinction, see [Handle errors and events](dialog-handle-errors-events.md).
-> - The `value` property of the `asyncResult` is set to a [Dialog](/javascript/api/office/office.dialog) object, which exists in the host page, not in the dialog box's execution context.
-> - The `processMessage` is the function that handles the event. You can give it any name you want.
-> - The `dialog` variable is declared at a wider scope than the callback because it is also referenced in `processMessage`.
+Office passes an [AsyncResult](/javascript/api/office/office.asyncresult) object to the callback. It represents the result of the attempt to open the dialog box. It does not represent the outcome of any events in the dialog box. For more on this distinction, see [Handle errors and events](dialog-handle-errors-events.md).
+
+    - The `value` property of the `asyncResult` is set to a [Dialog](/javascript/api/office/office.dialog) object, which exists in the host page, not in the dialog box's execution context.
+    - The `processMessage` is the function that handles the event. You can give it any name you want.
+    - The `dialog` variable is declared at a wider scope than the callback because it is also referenced in `processMessage`.
```

Comment:
```suggestion
- The `value` property of the `asyncResult` is set to a [Dialog](/javascript/api/office/office.dialog) object, which exists in the host page, not in the dialog box's execution context.
- The `processMessage` is the function that handles the event. You can give it any name you want.
- The `dialog` variable is declared at a wider scope than the callback because it is also referenced in `processMessage`.
```